classes/Tutor.php

```diff
@@ -560,11 +560,6 @@ public function __construct() {
 		 */
 		new Ecommerce();
 
-		/**
-		 * Tutor AI generation controller
-		 */
-		new MagicAI();
```

Comment:
@ahamed Vai, this seems irrelevant to this PR. Was it intentional or a accidental?

Reply:
Intentional.